src/core/lv_obj_tree.c

```diff
@@ -249,23 +287,24 @@ void lv_obj_swap(lv_obj_t * obj1, lv_obj_t * obj2)
     lv_event_send(parent2, LV_EVENT_CHILD_CHANGED, obj1);
 
     lv_obj_invalidate(parent);
-    if( parent != parent2) {
+
+    if(parent != parent2)
+    {
         lv_obj_invalidate(parent2);
     }
-
     lv_group_swap_obj(obj1, obj2);
 }
 
 void lv_obj_move_up(lv_obj_t * obj)
```

Comment:
Shouldn't  `lv_obj_move_up/down` be removed?

Reply:
I was not sure about removing these functions, the nice thing about them is that they will take care of calculating and testing the new position.
but i can remove them.

Reply:
If index is too small or too large lv_obj_move_to_index will return too.

Reply:
done

Reply:
Thanks!